async_grpc/rpc_handler.h

```diff
@@ -21,12 +21,12 @@
 #include "async_grpc/rpc.h"
 #include "async_grpc/rpc_handler_interface.h"
 #include "async_grpc/rpc_service_method_traits.h"
-
 #include "google/protobuf/message.h"
-
 #include "glog/logging.h"
-
 #include "grpc++/grpc++.h"
+#ifdef TRACING_SUPPORT
```

Comment:
In Cartographer, we use `#if USE_PROMETHEUS`.
ifdef has the disadvantage it is unclear when it is not set, and awkward to use in logic, so I slightly prefer if.